scyjava/convert/__init__.py

```diff
@@ -0,0 +1 @@
+from ._convert import *
```

Comment:
Why? I ask naively. I don't understand what the purpose is behind all these nested folders?

Reply:
Because I am also adding `_pandas.py` file in that same folder. I prefer to have many files with a small number of lines in it that a single file with thousands of lines. I feel it's also more Pythonic.

That being said if you have strong feelings about it, I can go back to a single file.